library/payfine/PayFineControl.java

```diff
@@ -50,18 +59,18 @@ public void Cancel() {
 		state = ControlState.CANCELLED;
 	}
 
-	public double payFine(double AmOuNt) {
+	public double payFine(double amount) {
 		if (!state.equals(ControlState.PAYING))
 			throw new RuntimeException("PayFineControl: cannot call payFine except in PAYING state");
 
-		double ChAnGe = member.payFine(AmOuNt);
-		if (ChAnGe > 0)
-			ui.display(String.format("Change: $%.2f", ChAnGe));
+		double change = member.payFine(amount);
+		if (change > 0)
+			ui.display(String.format("Change: $%.2f", change));
 
 		ui.display(member.toString());
 		ui.setState(PayFineUi.UiState.COMPLETED);
 		state = ControlState.COMPLETED;
-		return ChAnGe;
+		return change;
 	}
 
 }

```

Comment:
Follow the naming convention for the below highlighted names
private Library **LiBrArY**;
public void **CardSwiped**(int memberId)
public void **Cancel**()

Use a curly brace when yu define the Condition 
if (!state.equals(ControlState.READY))
if (!state.equals(ControlState.PAYING))
if (change > 0)

Reply:
Thanks for the feedback, I will make the changes